jfrog-applications/jfrog-cli/cli-for-jfrog-platform-services/platform-workers-init.md

```diff
@@ -0,0 +1,24 @@
+# Initialize a Worker
+
+## Overview
+
+This command is used to initialize a new platform worker.
```

Comment:
be more descriptive like "This command will create all necessary file in the current directory in order to start writing your worker : sample code, manifest file and sample tests" something like that